scripts/creation-portal-migration/SL-DataMapping.csv

```diff
@@ -0,0 +1,65 @@
+ENV,authorId,mappedUserId,userName,rootOrgId,rootOrgName,org_id,srcOrgAdminId,srcOrgAdminUserName,contributorOrgAdminId,contributorOrgAdminUserName,solutionId,solutionName,programId,programName
```

Comment:
@swetha-aggidevara we can also include a guide on how to generate this CSV, with help of Piyush if need be